CollAction/Dockerfile

```diff
@@ -1,4 +1,4 @@
-﻿FROM microsoft/aspnetcore:1.1.1
+﻿FROM microsoft/aspnetcore:1.1.2
```

Comment:
So we still need to build first on the host OS which is causing some issues. Is there an easy way to get a dev server working within docker too?

Reply:
No easy, but maybe we should make a build docker image